Author: Apollon Oikonomopoulos <apoikos@debian.org>
Description: Relax cabal build-dependencies
 Drop upper version limits for json and utf8-string. Upstream uses known-good
 version limits and sid currently has newer versions of json (0.9.1) and
 utf8-string (1). Since ganeti builds successfully with sid's versions, we
 remove the upper version limit.

Last-Update: 2015-07-21

--- a/cabal/ganeti.template.cabal
+++ b/cabal/ganeti.template.cabal
@@ -54,15 +54,15 @@
     , transformers                  >= 0.3.0.0
     , unix                          >= 2.5.1.0
 
-    , attoparsec                    >= 0.10.1.1   && < 0.13
+    , attoparsec                    >= 0.10.1.1   && < 0.14
     , base64-bytestring             >= 1.0.0.1    && < 1.1
     , case-insensitive              >= 0.4.0.1    && < 1.3
     , Crypto                        >= 4.2.4      && < 4.3
     , curl                          >= 1.3.7      && < 1.4
     , hinotify                      >= 0.3.2      && < 0.4
     , hslogger                      >= 1.1.4      && < 1.3
-    , json                          >= 0.5        && < 0.9
-    , lens                          >= 3.10       && < 4.8
+    , json                          >= 0.5
+    , lens                          >= 3.10
     , lifted-base                   >= 0.2.0.3    && < 0.3
     , monad-control                 >= 0.3.1.3    && < 1.1
     , MonadCatchIO-transformers     >= 0.3.0.0    && < 0.4
@@ -71,8 +71,8 @@
     , regex-pcre                    >= 0.94.2     && < 0.95
     , temporary                     >= 1.1.2.3    && < 1.3
     , transformers-base             >= 0.4.1      && < 0.5
-    , utf8-string                   >= 0.3.7      && < 0.4
-    , zlib                          >= 0.5.3.3    && < 0.6
+    , utf8-string                   >= 0.3.7
+    , zlib                          >= 0.5.3.3    && < 0.7
 
     -- Executables:
     -- , happy
